relay/sources/relayd/src/remote_run.rs

```diff
@@ -38,6 +38,7 @@ use std::{
     process::{Command, Stdio},
     str::FromStr,
     sync::Arc,
+    path::{Path, PathBuf},
```

Comment:
`Path` is not used and produces a warning, we should not have warnings in merged code.